wolfcrypt/src/ecc.c

```diff
@@ -4913,35 +4913,50 @@ int wc_ecc_sign_hash_ex(const byte* in, word32 inlen, WC_RNG* rng,
     #else
         mp_int* sign_k = NULL;
     #endif
+    #ifdef WC_ECC_NONBLOCK_ONLY
+        /* perform blocking call to non-blocking function */
+        ecc_nb_ctx_t nb_ctx;
```

Comment:
Compiled with just WC_ECC_NONBLOCK_ONLY and it didn't have the type defined.
Wasn't expecting to have to define WC_ECC_NONBLOCK as well.

Reply:
Ah yes... Thanks. Fix pushed.